pkg/minishift/registration/redhat.go

```diff
@@ -67,7 +72,9 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 			subscriptionCommand := fmt.Sprintf("sudo -E subscription-manager register --auto-attach "+
 				"--username %s "+
 				"--password '%s' ", param.Username, util.EscapeSingleQuote(param.Password))
+			s.Start()
 			_, err = registrator.SSHCommand(subscriptionCommand)
+			s.Stop()
```

Comment:
You need to add a new line as currently it looks like below. Also I would suggest that once registration completed, you should say so in the console and move to the next line. 

```
Red Hat Developers or Red Hat Subscription Management (RHSM) username: lmohanty@redhat.com
Red Hat Developers or Red Hat Subscription Management (RHSM) password: [HIDDEN]
/     Downloading OpenShift binary 'oc' version 'v1.5.1'
 19.96 MiB / 19.96 MiB [===============================================================================================================================================================================] 100.00% 0s
-- Checking OpenShift client ... OK
-- Checking Docker client ... OK
-- Checking Docker version ... OK
-- Checking for existing OpenShift container ... OK
-- Checking for openshift/origin:v1.5.1 image ... 
   Pulling image openshift/origin:v1.5.1
   Pulled 0/3 layers, 3% complete
   Pulled 1/3 layers, 76% complete
```